packs/mmonit/actions/list_status_hosts.py

```diff
@@ -0,0 +1,30 @@
+from lib.mmonit import MmonitBaseAction
+
+
+class MmonitListStatusHost(MmonitBaseAction):
+    def run(self, host_id=None, hostgroupid=None, status=None, platform=None, machine=None, led=None):
+        self.login()
+        data = {}
+
+        # Way too explicit for my taste but I guess its easier to understand
```

Comment:
To clarify it, that's not what I meant with explicit :)

With explicit, I simply meant explicitly declaring all the arguments in the method signature (which is indeed done here).